ajax/libs/redux-persist/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "redux-persist",
+  "version": "4.0.0-alpha6",
+  "description": "persist and rehydrate redux stores",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
+  "ava": {
+    "require": [
+      "babel-register"
+    ],
+    "babel": "inherit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rt2zz/redux-persist.git"
+  },
+  "homepage": "https://github.com/rt2zz/redux-persist",
```

Comment:
Please remove `homepage` field because we'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Thanks